utl/vw-hyperopt.py

```diff
@@ -219,11 +220,16 @@ def get_hyperparam_string(self, **kwargs):
     def compose_vw_train_command(self):
         data_part = ('vw -d %s -f %s --holdout_off -c '
                      % (self.train_set, self.train_model))
+        if self.labels_clf_count > 2: # multiclass, should take probabilities
+            data_part += ('--oaa %s --loss_function=logistic --probabilities '
```

Comment:
Would be nice to support other multiclass methods in addition to `--oaa`.
e.g.: `--ect`, `--recall_tree`, `--wap`, (with & without ldf) etc.
They all have similar interface (expect number of classes <k> as arg)